grafana-plugin/src/containers/UserSettings/UserSettings.module.css

```diff
@@ -8,3 +8,18 @@
   width: calc(100% - var(--spacing) * 2); /* allow lateral spacing */
   max-width: 1100px;
 }
+
+.alerts-container {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+  gap: 10px;
+
+  &:empty {
+    display: none;
+  }
+}
+
+.alert {
+  margin: 0;
+}
```

Comment:
@grafana/grafana-oncall-frontend this was essentially copy-pasta'd from [`grafana-plugin/src/containers/Alerts/Alerts.module.scss`](https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/containers/Alerts/Alerts.module.scss#L1-L19), not sure if we want to extract this out somehow?